src/Widgets/EntryPopover/Generic.vala

```diff
@@ -12,9 +12,10 @@ public abstract class Tasks.Widgets.EntryPopover.Generic<T> : Gtk.EventBox {
     public string placeholder { get; construct; }
     public T value { get; set; }
 
-    private Gtk.MenuButton popover_button;
     private T value_on_popover_show;
 
+    private Gtk.EventControllerMotion motion_controller;
```

Comment:
Do we keep this class being based on Gtk.EventBox until actual GTK4 migration?

(Both choices―keeping that and rebasing on another class while we're here―are fine, just I thought this class no longer need to be based on Gtk.EventBox since it now uses EventControllerMotion)

Reply:
In my experience box won't capture certain events in GTK3 even with the motion controller, so I just left it as event box. It looks like it's working here though, so changed!